src/adminPartMain/kotlin/TestsAssociatedWithBuildStorage.kt

```diff
@@ -26,22 +26,45 @@ class MutableMapTestsAssociatedWithBuild : TestsAssociatedWithBuild {
         }
     }
 
-    private fun testsAssociatedWithBuild(buildVersion: String): List<TypedTest> {
-        return map[buildVersion]?.flatMap { it -> it.tests }.orEmpty()
+    private suspend fun enabledFinishedScopes(
```

Comment:
smth quiet familiar is happening in calculateAndSendBuildCoverage(..), have you seen that?